app/code/Magento/Sales/Setup/UpgradeData.php

```diff
@@ -118,4 +162,34 @@ private function convertSerializedDataToJson($setupVersion, SalesSetup $salesSet
         }
         $this->aggregatedFieldConverter->convert($fieldsToUpdate, $salesSetup->getConnection());
     }
+
+    /**
+     * Fill quote_address_id in table sales_order_address if it is empty.
+     */
+    public function fillQuoteAddressIdInSalesOrderAddress()
+    {
+        $addressCollection = $this->addressCollectionFactory->create();
+        /** @var \Magento\Sales\Model\Order\Address $orderAddress */
+        foreach ($addressCollection as $orderAddress) {
```

Comment:
@RomaKis Whole collection will be iterated without any limits.  BatchIteration at least should be used. Or it should be upgraded via query not over iterating every adress, loading it and saving.

Reply:
@valdislav you can create pull request with this fixed.